lib/github_service/concerns/is_team_member.rb

```diff
@@ -0,0 +1,32 @@
+module IsTeamMember
+  extend ActiveSupport::Concern
+
+  def triage_member?(username)
+    self.class.triage_team_members.include?(username)
+  end
+
+  module ClassMethods
+    # List of usernames for the traige team
+    def triage_team_members
+      return @triage_team_members if defined? @triage_team_members
```

Comment:
We probably need to bust this cache occasionally to bring in anyone that was recently added to or removed from the team, right?

Reply:
Probably, but this is where I don't have much insight into how the bot is run, since periodic worker restarts pretty much "could" handle that.  But I am not sure how often that happens.

We could also just bust the cache every 24 hours as well.

Reply:
Resolved.  See https://github.com/ManageIQ/miq_bot/pull/482#issuecomment-597930848